### DISCUSSION TITLE
Starting VS code in wsl not working

### DISCUSSION THREAD

**Discussion by Aishwaria8786** [Cyfrin/foundry-full-course-cu#280] [Q&A] [answered]
![Screenshot (264)](https://github.com/Cyfrin/foundry-full-course-f23/assets/68544977/4d4e4e74-fb84-4381-8d4d-d8a46d8ed798)
Can anyone help me regarding this issue?

**Comment by Naman1729** [comment]
Hey @Aishwaria8786 , have you install **Ubuntu** .

**Reply by Aishwaria8786** [reply]
Yes. It was working fine yesterday but now it is showing this issue.

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Hey @Aishwaria8786, refer to these links and try to attempt the solutions provided.

https://stackoverflow.com/questions/56972740/how-can-i-fix-the-segmentation-fault-error-in-visual-studio-codes-terminal

https://github.com/microsoft/vscode-remote-release/issues/3117

**Reply by Aishwaria8786** [reply]
Thanks a lot. It solved the issue.
